mitodl/ocw-studio
# Blank _logo.html to remove default logo image

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #996 

#### What's this PR do?
Adds a blank `_logo.html` template file so emails won't show the default image logo.

#### How should this be manually tested?
- Set the following in your .env file:
  ```
    MITOL_MAIL_FROM_EMAIL: ocw-prod-support@mit.edu
    MITOL_MAIL_REPLY_TO_ADDRESS: ocw-prod-support@mit.edu
  ```
- Create a user with a real email address you can check.
- Run this in a shell:
```
from websites.api import *
mail_on_publish(<your_website.name>, "live", True, <your_user_id>)
```

Wait for a new email to arrive, it should have no embedded image and the From and Reply-To fields should be `ocw-prod-support@mit.edu`


#### Screenshots (if appropriate)
<img width="442" alt="Screen Shot 2022-02-11 at 2 36 26 PM" src="https://user-images.githubusercontent.com/187676/153658072-e2976224-8e35-48bf-9123-0d22194cdfc8.png">



<img width="917" alt="Screen Shot 2022-02-11 at 2 36 08 PM" src="https://user-images.githubusercontent.com/187676/153658074-bce3e522-2fd1-4476-affc-b550ca868816.png">
